soh/soh/OTRGlobals.cpp

```diff
@@ -1186,14 +1189,7 @@ extern "C" SkeletonHeader* ResourceMgr_LoadSkeletonByName(const char* path) {
 }
 
 extern "C" s32* ResourceMgr_LoadCSByName(const char* path) {
-    // Handle mq vs nonmq for cutscenes due to scene/ paths
-    auto res = ResourceMgr_LoadResource(path);
-
-    if (res == nullptr) {
-        return nullptr;
-    }
-
-    return (s32*)res->GetPointer();
+    return (s32*)GetResourceDataByNameHandlingMQ(path);
```

Comment:
everything here is now happening in `GetResourceDataByNameHandlingMQ`, this makes sense